app/mailers/nsm/feedback_messages/feedback_base.rb

```diff
@@ -58,7 +58,7 @@ def ufn
       end
 
       def claim_total
-        @submission.data['submitted_total_inc_vat'] || @submission.data['submitted_total'] || 0
```

Comment:
This field was never populated. I'm replacing it with a formatted string - I don't know if that will be a problem in Notify when this value is interpolated - if so we'll need to rewrite CoreCostSummary to give access to the raw numbers after it does the calculation of totals.